mesh/mesh.hpp

```diff
@@ -1352,10 +1352,22 @@ std::ostream &operator<<(std::ostream &out, const Mesh &mesh);
     Mesh. See Mesh::GetGeometricFactors(). */
 class GeometricFactors
 {
+
+protected:
+   /// Element restricted nodes
+   /// [TODO move to stack when pools are available]
+   /** This array uses a column-major layout with dimensions (ND x SDIM x NE) where
+        - ND   = number of degrees of freedom per element,
+        - SDIM = space dimension of the mesh = mesh.SpaceDimension(), and
+        - NE   = number of elements in the mesh. */
+   Vector Enodes;
+
 public:
-   const Mesh *mesh;
-   const IntegrationRule *IntRule;
+   const Mesh *mesh {nullptr};
```

Comment:
I like these initialization too, but it is usually done through the constructors.

Reply:
Done